documentation/assertions/any/to-satisfy.md

```diff
@@ -15,14 +17,18 @@ expect({ hey: { there: true } }, 'to exhaustively satisfy', {
 });
 ```
 
-Regular expressions and functions in the right-hand side object will be run
-against the corresponding values in the subject:
+Regular expressions and `expect.it` expressions in the right-hand side object
+will be run against the corresponding values in the subject:
```

Comment:
Hmm, would be good with an example of an `expect.it` below as well, since it's being mentioned.

Reply:
Counterproposal - I think we should stop mentioning `expect.it` here and instead defer introducing that until the new dedicated section for it below. What do you think?

Reply:
Fine :+1:

Reply:
I've replaced this with a mention that additional syntax exists and linked to the complex specifications section using an anchor link.